tutorials/whisper_e2e.py

```diff
@@ -160,30 +163,31 @@ def preprocessing(audio_data):
             "audio_pcm": {1: "sample_len"},
         }
     )
-    onnx.save_model(pre_model, prep_model_name)
+    onnx.save_model(pre_model, os.path.join(root_dir, prep_model_name))
     if USE_ONNX_STFT:
         pre_model = _to_onnx_stft(pre_model)
 
-    pre_f = PyOrtFunction.from_model(pre_model)
+    pre_f = PyOrtFunction.from_model(pre_model, cpu_only=True)
```

Comment:
Did you encounter any issue if no cpu_only specified, or just be safe?

Reply:
If `cpu_only` is not specified, there is a CUDA out of memory error that occurs.